### ISSUE TITLE
Stale documentation of `terraform validate` command

### ISSUE THREAD

**Issue by cpxPratik** [hashicorp/terraform-website#874]
Documentation for command `terraform validate` on https://www.terraform.io/docs/commands/validate.html is not up-to-date with Terraform v0.12.x.

There are no extra flags other than `-json` when doing `terraform --help validate`

**Comment by nfagerlund** [comment]
Good catch, thank you. I just did some testing: 

- `-no-color` is still real. 
- `-check-variables` blows up.
- `-var` and `-var-file` are still _allowed,_ but they don't seem to do anything, and validate no longer seems to do anything with variable _values._ (It does check to make sure the referencing context allows/expects a value of the variable's declared type; for example, when I tried using a `list` variable in a `null_resource`'s `triggers`, it blew up at me.)
